substrate/frame/merkle-mountain-range/src/tests.rs

```diff
@@ -787,24 +787,3 @@ fn does_not_panic_when_generating_historical_proofs() {
 		);
 	});
 }
-
-#[test]
-fn generating_and_verifying_ancestry_proofs_works_correctly() {
```

Comment:
why remove this test?

Reply:
It was very similar with the new test added to `beefy_mmr`. But it's good to have it. I added it back.